frontend/app/work_packages/services/work-package-field-service.js

```diff
@@ -146,7 +146,11 @@ module.exports = function(
     });
 
     if (!WorkPackageFieldService.isRequired(workPackage, field)) {
-      var arrayWithEmptyOption = [{ href: null }];
+      var nullValueLabel = I18n.t('js.inplace.null_value_label');
+      var arrayWithEmptyOption = [{
+        href: null,
+        name: nullValueLabel
```

Comment:
this is a leftover (I originally set the `name` and the `title`). I guess it'll still help with line length.